README.md

```diff
@@ -414,19 +414,17 @@ set PATH=C:\STL\out\x64\out\bin\amd64;%PATH%
 You don't need any extra steps to run with test code and the code in STL headers instrumented with [ASan][].
 The test matrices include both ASan and non-ASan configurations.
 
-However, to instrument the separately-compiled code (the DLL, the satellites, the [Import Library][] - everything that's
-in `/stl/src`), you need to build the STL with ASan. Change the build steps to add `-DSTL_ASAN_BUILD=ON`:
+However, to instrument the separately-compiled code (the DLL, the satellites, the [Import Library][] - everything that's in `/stl/src`), you need to build the STL with ASan. Change the build steps to add `-DSTL_ASAN_BUILD=ON`:
 
 ```
 cmake --preset x64 -DSTL_ASAN_BUILD=ON
 cmake --build --preset x64
 ```
 
-ASan-instrumented STL binaries require that the executable be instrumented as well, so you'll have to skip the non-ASan
-configurations by passing `-Dtags=ASAN` to `stl-lit.py`:
+ASan-instrumented STL binaries require that the executable be instrumented as well, so you'll have to skip the non-ASan configurations by passing `-Dtags=ASAN` to `stl-lit.py`:

```

Comment:
This line looks overlong; you should probably revert this bit.

Reply:
Reverted!